### PR TITLE
[RFR] Preparations for new graphics3

### PR DESCRIPTION
**Pull request by touv**

### CODE REVIEW

File: src/api/controller/api/run.js [hunk 1]
```diff
@@ -48,6 +48,15 @@ export const runRoutine = async (ctx, routineCalled) => {
         host: ctx.host,
         hostname: ctx.hostname,
     };
+    const fields = ctx.path
```
Comment (Kmaschta) [inline src/api/controller/api/run.js]: Manually parse the request URL isn't recommended.
It's better to pass all the params from the router IMHO

```js
export const runRoutine = async (ctx, routineCalled, field1, field2) => {
    // ...
    const fields = [routineCalled, field1, field].filter(x => x);
}
```

Comment (touv) [reply src/api/controller/api/run.js]: Yes, you're right. 
However, some older routines use ctx.path to detect the concerned fields.
I'm afraid to break something, I propose to change nothing to ensure backward compatibility

Comment (touv) [reply src/api/controller/api/run.js]: oh... I understood, it's a new variable so ctx.path is still accessible... i'll patch

File: src/api/statements/LodexRunQuery.js [hunk 2]
```diff
@@ -13,7 +13,7 @@ export const createFunction = mongoClientImpl =>
         }
         const from = this.getParam('from', data.$from || 'dataset');
         const query = this.getParam('query', data.query || {});
-        const limit = this.getParam('limit', data.limit || 10);
+        const limit = this.getParam('limit', data.limit || 1000000);
```
Comment (Kmaschta) [inline src/api/statements/LodexRunQuery.js:16]: Is this the real default value or just to debug?
If you confirm it's the correct value, I'll merge! :)

**Review by Kmaschta — APPROVED** [review]

File: src/api/statements/LodexRunQuery.js (hunk 2 above)
Comment (touv) [reply src/api/statements/LodexRunQuery.js:16]: yes, it's the correct value

**Pull request merged**